### PR TITLE
fixes #129 custom start command history

### LINKED ISSUE #129: Liberty: History action to save previous Start... choices
To avoid having to re-type the same custom parameters each run add a "History" option similar to VS Code for Maven to view past runs

image

image

### PR DESCRIPTION
**Pull request by travis1111**
Save custom start command params.

### CODE REVIEW

File: src/liberty/dashboard.ts [hunk 1]
```diff
@@ -9,13 +9,34 @@
...
+      }
+    }
+    if ( startCommandParams !== undefined ) {
+      for ( let i = 0; i < startCommandParams.length; i++ ) {
+        this.lastUsedStartParams.push(startCommandParams[i]);
+      }
+    }
+  }
+
+
```
Comment (TrevCraw) [inline src/liberty/dashboard.ts:31]:
```suggestion
// add last used custom parameters to the top of the list
```

**Comment by TrevCraw** [comment]
@travis1111 would you like to incorporate @cherylking's suggestion to show the base command when hovering over the top selection into this PR or would you like a separate issue opened to look into it?

File: src/liberty/dashboard.ts (hunk 1 above)
Comment (travis1111) [reply src/liberty/dashboard.ts]: addressed

**Comment by travis1111** [comment]
> @travis1111 would you like to incorporate @cherylking's suggestion to show the base command when hovering over the top "empty" selection into this PR or would you like a separate issue opened to look into it?

Please open a separate issue.  Thanks!

**Review by kathrynkodama — APPROVED** [review]

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**